client/src/Containers/Addtitle.jsx

```diff
@@ -1,59 +1,119 @@
+      <Dialog 
+        open={open} 
+        onClose={ () => setOpen(false)}
+        maxWidth="md"
+        >
 
-      
-      {/* -------------------------------Title input text field---------------------------------------------------------- */}
-      <Grid container alignItems="center">
+
+        <DialogTitle style={{ marginLeft : '10px' , fontSize : '1.2em'}}> Add the Title</DialogTitle>
+
+        
+        {/* -------------------------------Title input text field---------------------------------------------------------- */}
+        
           <TextField
             required
             id="title-input"
-            className={classes.questioninput}
+            fullWidth
             label="Type the title of Quiz here"
             variant="outlined"
+            style={{
+              width : '900px',
+              margin : '10px 30px'
+            }}
             />
-      </Grid>
-
-      {/* -------------------------------Quiz Link Button---------------------------------------------------------- */}
-      <Grid container alignItems="center">
-            <Button
-              variant="contained"
-              style={{
-                padding: '12px',
-                margin: '20px auto'
-              }}
-              >
-              Get Quiz Link
-            </Button>
-      </Grid>
+        
+
+        {/* -------------------------------Quiz Link Button---------------------------------------------------------- */}
+        
+            <DialogActions>
+              <Button
+                variant="contained"
+                style={{
+                  padding: '12px',
+                  margin: '20px auto'
+                }}
+                onClick={() => sendToDB()}
+                >
+                Get Quiz Link
+              </Button>
+
+            </DialogActions>
+        
+      </Dialog>
+
+      {/* -------------------------------Dialog for Quiz Code---------------------------------------------------------- */}
+      <Dialog
+        open={openQuizLink}
+        onClose={() => setOpenQuizLink(false)}
+        >
+        <DialogTitle> Here is your Quiz Link</DialogTitle>
+        
+        <DialogContent>
+          <TextField
+            disabled
+            value={quizLink}
+            style={{
+              width : '300px',
+              margin : '10px auto'
+            }}
+          />
+        </DialogContent>
+
+        <DialogActions>
+          <Button
+            variant="contained"
+            onClick={() => navigator.clipboard.writeText(quizLink)}
+          >
+            Copy to Clipboard
+          </Button>
+        </DialogActions>
+      </Dialog>
       
     </div>
   );
 };
 
-export default Addquestion;
+Addtitle.propTypes = {
+  open : PropTypes.bool.isRequired,
+  setOpen : PropTypes.func.isRequired,
+  data : PropTypes.shape.isRequired,
+  setData : PropTypes.func.isRequired
+}

```

Comment:
It's good that you have added PropTypes. 👍

Reply:
Thanks😊